### ISSUE TITLE
TypeError: Cannot read property 'json' of undefined

### ISSUE THREAD

**Issue by koooge** [vlucas/frisby#456]
Hi. I got error

```package.json
   "frisby": "^2.0.14",
    "jest": "^22.4.3",
```

```
    TypeError: Cannot read property 'json' of undefined
      
      at node_modules/frisby/src/frisby/spec.js:261:74
```

I doubt that _response(FrisbyResponse) becomes undefined when FrisbySpec throws Error before `new FrisbyResponse`
https://github.com/vlucas/frisby/blob/master/src/frisby/spec.js#L127-L133

**Comment by koooge** [comment]
This issue seems to occur  in case of server not response.
But this error shows frisby cannot handle the timeout error correctly.

refs? #429

**Comment by koooge** [comment]
#447

**Comment by vlucas** [comment]
Released `v2.0.15` with this fix. Please upgrade.
